lib/enigma.rb

```diff
@@ -36,8 +36,12 @@ def encrypt_message(message, key, date)
     shifts = create_shifts(key, date)
     split_message(message).each do |chars|
       chars.zip(shifts).each do |char, shift_value|
-        new_index = (@alphabet.find_index(char) + shift_value) % 27
-        encrypted.concat(@alphabet[new_index])
+        if alphabet.include?(char)
+          new_index = (@alphabet.find_index(char) + shift_value) % 27
+          encrypted.concat(@alphabet[new_index])
+        else
+          encrypted.concat(char)
+        end
```

Comment:
This if/else accounts for characters not in the alphabet. I am planning on refactoring this whole method to something more readable/ shorter.